src/presentation/routers/articles-router.ts

```diff
@@ -0,0 +1,175 @@
+import express, { Request, Response } from 'express';
+import multer from 'multer';
+import path from 'path';
+import fs from 'fs';
+
+import HttpStateCodes from '../../utils/http-state-codes';
+import NoSQLWrapper from '../../data/interfaces/data-sources/no-sql-wrapper';
+import { ValidateFields } from '../../interceptors/Validate-fields';
+import { AccessControlMiddleware, ValidateAuthentication } from '../../interceptors/Access-auth-validate';
+
+export const ArticlesRouter = (db: NoSQLWrapper) => {
+    // routing
+    const router = express.Router();
+
+      const storage = multer.diskStorage({
+        destination: (req, file, cb) => {
+          const uploadPath = 'uploads';
```

Comment:
Ok, this is another, don't hardcode the paths, you need to get this data from the environments.
example
```
  process.env.API_UPLOAD_DIR
```